backends/kv/writer.go

```diff
@@ -61,14 +63,46 @@ func (kv *Backend) Write(request *frames.WriteRequest) (frames.FrameAppender, er
 		return nil, err
 	}
 
+	var schema v3ioutils.V3ioSchema
+	schema, schemaError := v3ioutils.GetSchema(tablePath, container)
+
+	// Ignore 404 error, since it makes sense there is no schema yet.
+	tableAlreadyExists := true
+	if err != nil {
```

Comment:
Redundant if - `err` cannot be anything but `nil` at this point.

Reply:
I guess you meant to check and use `schemaError`. Better just overwrite `err` I guess, to avoid this kind of confusion.